mkdocs_hook.py

```diff
@@ -0,0 +1,194 @@
+import logging
```

Comment:
Can we move this file into .github dir so it does not clutter the top of the repo.  Sample for mkdocs.yml if possible.

Reply:
Let me test if it works that way.